tests/config_test.py

```diff
@@ -55,7 +55,7 @@ def test_load_minigraph(self, get_cmd_module, setup_single_broadcom_asic):
             traceback.print_tb(result.exc_info[2])
             assert result.exit_code == 0
             assert "\n".join([l.rstrip() for l in result.output.split('\n')]) == load_minigraph_command_output
-            assert mock_run_command.call_count == 7
+            assert mock_run_command.call_count == 8
```

Comment:
>call_count [](http://example.com/codeflow?start=36&length=10)

Could you test your feature/fix in this test case?

Reply:
Sure. Added a few checks in this test case, please help review.